### DISCUSSION TITLE
Is there any way to override the default exclude filters?

### DISCUSSION THREAD

**Discussion by msellitto** [universal-ctags/ctags#3947] [Q&A] [answered]
I'm attempting to generate ctags for a perl project I'm working on. One of the files has the extension of .lib (it's basically a Perl module, don't ask me why it's named with that extension though, it's a really old project).

Whenever I try to generate tags for it I see that ctags excludes *.lib as one of the default exclusion extensions. I try using --langmap or --map-perl that I want .lib extensions to be Perl it still doesn't create the tags. It still says it's ignoring that file. Is there any way to override this? 

Thanks.

**Comment by masatake** [comment, marked as answer]
`--exclude-exception` is the option for you.

```
$ cat /tmp/foo.lib 
#!/bin/perl

sub f()
{
}
$ ctags -o - --exclude-exception='*.lib' --map-Perl=+.lib /tmp/foo.lib
f	/tmp/foo.lib	/^sub f()$/;"	s
```

**Reply by msellitto** [reply]
Thanks, I was using an older version of universal ctags that didn't have that option. Updated it to the latest and things are working!
